DiamondLightSource/dodal
# Add Bimorph Mirrors, Slits

Add Bimorph Mirrors, Slits Ophyd Devices

### Instructions to reviewer on how to test:
1. Create + activate venv

To test slits:
1. Start the slit simulator (ask Daniel)
2. Run `pytest tests/devices/unit_tests/test_slit_interface.py`
3. Ensure all tests pass

To test bimorphs:
1. SSH into BL02J
2. Run `pytest tests/devices/unit_tests/test_CAENels_bimorph_mirror_8_channel.py`
3. Ensure all tests pass

### Checks for reviewer
- [ ] Would the PR title make sense to a scientist on a set of release notes
- [ ] If a new device has been added does it follow the [standards](https://github.com/DiamondLightSource/dodal/wiki/Device-Standards)
